<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>amcpp</class>
 <widget class="QMainWindow" name="amcpp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>amcpp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="Phonon::SeekSlider" name="seekSlider">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>431</width>
      <height>24</height>
     </rect>
    </property>
    <property name="iconVisible">
     <bool>true</bool>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Phonon::VolumeSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>641</width>
      <height>321</height>
     </rect>
    </property>
    <property name="uniformRowHeights">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="headerCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="headerDefaultSectionSize">
     <number>200</number>
    </attribute>
    <attribute name="headerHighlightSections">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Artist</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>URL</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>421</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionPlay">
   <property name="text">
    <string>play</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
